.github/workflows/scripts/detect-all-changes.sh

```
@@ -0,0 +1,175 @@
+
+# Get current versions
+CORE_VERSION=$(cat core/version)
+FRAMEWORK_VERSION=$(cat framework/version)
+TRANSPORT_VERSION=$(cat transports/version)
+
+echo "📦 Current versions:"
+echo "   Core: $CORE_VERSION"
+echo "   Framework: $FRAMEWORK_VERSION"  
+echo "   Transport: $TRANSPORT_VERSION"
+
+START_FROM="none"
```

Shellcheck (0.10.0)

[warning] START_FROM appears unused. Verify use (or export if used externally).

(SC2034)